doc/datasets/index.rst

```diff
@@ -254,6 +254,36 @@ features::
 
  _`Faster API-compatible implementation`: https://github.com/mblondel/svmlight-loader
 
+.. _external_datasets:
+
+Loading from external datasets
+==============================
+
+scikit-learn works on any numeric data stored as a numpy array or a pandas DataFrame, 
+and there are many different ways to load your data into a usable format depending on
+the file type and the format of the file.
+ 
+Here are some recommended ways to load standard columnar data into a 
+format usable by scikit-learn 
+
+* `pandas.io <http://pandas.pydata.org/pandas-docs/stable/io.html>`_ to load heterogeneously typed data from various file formats and database protocols that can slice and dice before conversion to numerical features in a numpy array.
```

Comment:
While we are not as strict in docs as in code, please keep close to an 80-char line width. It makes this much more readable in text form.